__tests__/reactTags.test.js

```diff
@@ -538,10 +538,10 @@ describe('Test ReactTags', () => {
       );
       const $input = $el.find('.ReactTags__tagInputField');
 
-      $input.simulate('change', { target: { value: 'Ea' } });
+      $input.simulate('change', { target: { value: 'Or' } });
```

Comment:
revert this to `Ea` and change the expected result in line 544 instead

Reply:
Accepted